src/main/scala/io/iohk/ethereum/blockchain/sync/FastSync.scala

```diff
@@ -333,6 +333,10 @@ trait FastSync {
           bestBlockHeaderNumber = lastHeader.number
         }
       }
+
+      val blockHashes = blockHeadersObtained.map(_.hash)
+      self ! FastSync.EnqueueBlockBodies(blockHashes)
+      self ! FastSync.EnqueueReceipts(blockHashes)
```

Comment:
yes it is, we are testing if FastSync finishes and switches to regular and this is required (downloading block bodies and receipts)